assets/translations/en-US.json

```diff
@@ -110,3 +110,4 @@
   "Waiting_for_user_to_get_connected_to_WiFi": "Waiting for user to get connected to WiFi",
   "Can't_add_unsupported_device": "Can't add unsupported device"
 }
+
```

Comment:
Can you please not change the en-US.json file?

Reply:
sorry, i thought I rebased the repository to include original en-US.json file, but seemed to have cloned old files in my repository.  I think i fixed the issue.

Reply:
Not fixed yet, there is change in 2 files